docs/getting_started/webassembly.md

```diff
@@ -20,3 +20,19 @@ const wasmInstance = new WebAssembly.Instance(wasmModule);
 const main = wasmInstance.exports.main as CallableFunction
 console.log(main().toString());
 ```
+
+And for files:
+
+<!-- deno-fmt-ignore -->
```

Comment:
Why are we ignoring formatting?

Reply:
Not sure why, I just followed what the other code block did.

Reply:
After checking, it turns out that it was unnecessary, and that the other codeblock had a fmt ignore there because it included an array. I've made a commit that removes it.